src/libPMacc/include/mpi/MPIReduce.hpp

```diff
@@ -52,6 +52,7 @@ struct MPIReduce
     {
         if (isMPICommInitialized)
         {
+            /* exception inside the destructor are forbidden */
```

Comment:
They are not really forbidden, but should be assumed forbidden as a general rule because throwing exceptions in the destructor is even worse then goto ;-)